build.gradle.kts

```diff
@@ -16,14 +16,15 @@ buildscript {
 
 plugins {
     kotlin("jvm") version "1.3.72" apply false
-    id("org.jetbrains.dokka") version "0.10.0"
+    id("org.jetbrains.dokka") version "1.4.30"
```

Comment:
Why this older version of Dokka rather than the latest?

Reply:
I will confirm if that works as expected with v1.7.10 and will push the change accordingly

Reply:
it works well, although the documentation layout is significantly different.